criu/arch/x86/include/asm/compat.h

```diff
@@ -10,6 +10,7 @@
 #endif
 
 #include <sys/mman.h>
+#include "flog.h"
```

Comment:
Try to avoid header into header inclusion. Here flog.h is not needed in compat.h itself, but is apparently used to fanout itself into other .c files. Better patch those .c files.

Reply:
Sure sir, I will make this change and keep in mind for the future too.